src/content/configuration/stats.md

```diff
@@ -146,6 +146,9 @@ module.exports = {
     // Show dependencies and origin of warnings/errors (since webpack 2.5.0)
     moduleTrace: true,
 
+    // Show outputPath
+    outputPath: true | false,
```

Comment:
@EugeneHlushko Does this mean what you say?

Reply:
Yes